build.gradle

```diff
@@ -103,6 +108,10 @@ task afterEclipseBuildshipImport(type: Exec) {
 }
 
 
+googleJavaFormat {
```

Comment:
One blank line before this block, since it is within the same major section of the file.